ios/Sources/Component/Card/LargeCard.swift

```diff
@@ -29,18 +29,20 @@ public struct LargeCard: View {
     }
 
     public var body: some View {
-        VStack(alignment: .leading, spacing: 13) {
+        VStack(alignment: .leading, spacing: 0) {
```

Comment:
It seems better to use spacing for 16 & remove `.padding(.bottom, 16)`.
In Addition, wrap `Text(title)` and `HStack(spacing: 8)` with new `VStack(spacing: 13)`
to follow the current way that this components creates.